tools/gradle-plugin/src/test/kotlin/io/github/composegears/valkyrie/gradle/Utils.kt

```diff
@@ -0,0 +1,96 @@
+@file:OptIn(ExperimentalPathApi::class)
+
+package io.github.composegears.valkyrie.gradle
+
+import assertk.Assert
+import assertk.assertions.support.fail
+import java.nio.file.Path
+import java.nio.file.Paths
+import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.absolutePathString
+import kotlin.io.path.copyToRecursively
+import kotlin.io.path.createDirectories
+import kotlin.io.path.exists
+import kotlin.io.path.writeText
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.gradle.testkit.runner.TaskOutcome.SUCCESS
+import org.junit.jupiter.api.Assumptions.assumeFalse
+import org.junit.jupiter.api.Assumptions.assumeTrue
+
+internal fun buildRunner(root: Path, androidHome: Path? = null) = GradleRunner
```

Comment:
All `androidHome` is generated from `androidHomeOrSkip`, no need to leave this param.